CHANGELOG.md

```diff
@@ -47,6 +47,7 @@ next release
 * Updating all-in-one path ([@bigfleet](https://github.com/bigfleet) in [#4234](https://github.com/jaegertracing/jaeger/pull/4234))
 * Migrate the use of fsnotify to fswatcher in cert_watcher.go ([@haanhvu](https://github.com/haanhvu) in [#4232](https://github.com/jaegertracing/jaeger/pull/4232))
 * Restore baggage support in HotROD 🚗 ([@yurishkuro](https://github.com/yurishkuro) in [#4225](https://github.com/jaegertracing/jaeger/pull/4225))
+* Remove TerminateAfter for elastic queries ([@jakob3xd](https://github.com/Jakob3xD)) in [#4336](https://github.com/jaegertracing/jaeger/pull/4336)
```

Comment:
Thanks for thinking about adding this, but it's not necessary. We have scripts that will generate the changelog on each release. Please remove this entry.